src/Microsoft.TemplateEngine.Edge/Installers/NuGet/NuGetManagedTemplatePackage.cs

```diff
@@ -123,7 +123,7 @@ public DateTime LastChangeTime
             }
         }
 
-        public bool LocalPackage
+        public bool IsLocalPackage
```

Comment:
should we store value in property instead and have `Details` reading from it?

Reply:
I'd prefer to keep it reading from `Details` as it would need to check each write to `Details` (that can now happen even outside of the class scope)